helios-client/src/test/java/com/spotify/helios/client/AuthenticatingHttpConnectorTest.java

```diff
@@ -139,6 +140,20 @@ protected boolean matchesSafely(final URI item) {
     };
   }
 
+  private CustomTypeSafeMatcher<Map<String, List<String>>> hasKeys(final List<String> keys) {
```

Comment:
I'm going to skip this one; happy to learn more about it, but feels unnecessary for the moment.